src/tests/networkcontrol/NetworkControl_Reload_001.test.js

```diff
@@ -0,0 +1,41 @@
+import {
+  pluginDeactivate,
+  pluginActivate,
+  reloadNetworkAdapter,
+} from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+
+export default {
+  title: 'NetworkControl - Reload 001',
+  description: 'Check the Reload Functionality of Network Control Module',
+  steps: [
+    {
+      description: 'Check if NetworkControl Plugin is stopped correctly',
+      test: pluginDeactivate,
+      params: constants.networkControlPlugin,
+      assert: 'deactivated',
+    },
+    {
+      description: 'Check if NetworkControl Plugin is started correctly',
+      test: pluginActivate,
+      params: constants.networkControlPlugin,
+      assert: 'activated',
+    },
+    {
+      description: 'Invoke Reload',
+      sleep: 5,
+      test() {
+        return reloadNetworkAdapter.call(this, 'eth0')
+      },
+      validate(res) {
+        //TODO - Update validation by confirming that reload happened
```

Comment:
Same here, should we just check if the device is still reachable? Assuming it did not get another IP :)